sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/red_team/__init__.py

```diff
@@ -9,7 +9,7 @@
     from ._red_team_result import RedTeamResult
 except ImportError:
     print(
```

Comment:
Can we raise Import error instead of print statement ? but keep this message when import error is raised